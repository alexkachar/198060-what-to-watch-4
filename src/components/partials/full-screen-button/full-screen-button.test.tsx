import * as React from 'react';
import * as renderer from 'react-test-renderer';
import FullScreenButton from './full-screen-button';

it(`PlayButton renders correctly`, () => {
  const tree = renderer
    .create(
        <FullScreenButton
          onFullScreenButtonClick={jest.fn()}
        />, {
          createNodeMock: () => document.createElement(`div`)
        })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
