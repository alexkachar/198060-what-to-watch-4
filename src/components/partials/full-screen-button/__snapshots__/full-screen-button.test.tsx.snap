// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayButton renders correctly 1`] = `
<button
  className="player__full-screen"
  onClick={[MockFunction]}
  type="button"
>
  <svg
    height={27}
    viewBox="0 0 27 27"
    width={27}
  >
    <use
      xlinkHref="#full-screen"
    />
  </svg>
  <span>
    Full screen
  </span>
</button>
`;
